app/controllers/application_controller.rb

```diff
@@ -32,4 +32,36 @@ def set_current_user
     User.reset_current_user
     reset_session
   end
+
+  def set_events
+    @events = Event.on_user.show.order(date: :asc).limit(10)
+  end
+
+  def set_new_task
+    @task = Task.new
+  end
+
+  def set_search
+    @search = Task.search(params[:q])
+  end
+
+  def check_events
+    current_user.google_accounts.each do |account|
+      account.google_calendars.sync.each do |google_calendar|
+        if google_calendar.updated_at < Time.zone.yesterday
```

Comment:
__[rubocop]__
* Style/Next: Use `next` to skip iteration. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)